### DISCUSSION TITLE
Google Material Symbol Font

### DISCUSSION THREAD

**Discussion by Bodyclock-Games** [stride3d/stride#1952] [Q&A]
Hi All, I'm building a custom UI button that uses the Google Material Symbol Font for the icons. I'm accessing each icon/character with the Unicode char code. This all works fine with a standard font such as Arial but I get the following error/warning with the Material font.

`[SpriteFont]: Warning: Character 'εÇÖ' is not available in the static font character map`

I'm guessing the Unicode character is outside the standard set rasterised by Stride (using SDF). Is this a limitation of the rasterisation engine or is there a setting on the SpriteFont asset i can change?

Also, can I get custom UI elements to show in the editor?

Cheers Chris

**Comment by tebjan** [comment]
I think you can set the character range you want to prebuild for the SDF font at the SDF font asset properties.

**Reply by Bodyclock-Games** [reply]
Thanks for the suggestion. I got it to work using the character regions in the SpriteFont asset, which is great but not really practical in my case. When I use the Character Set option I add the required characters to NotePad with the font set to Material Icons. I cut and paste the required characters from Character Map in an undelimited string and save a txt file, but this doesn't seem to work. The project takes forever to build as if it's rendering every character. 
Do you know if is this the correct format for the Character Set text file?

I should add the project finished building after 7 minutes and the character didn't appear.
